lighthouse-core/lib/i18n/i18n.js

```diff
@@ -277,6 +278,20 @@ function getFormatted(icuMessageIdOrRawString, locale) {
   return icuMessageIdOrRawString;
 }
 
+/**
+ * @param {LH.Locale} locale
+ * @param {string} icuMessageId
+ * @param {*} [values]
+ * @return {string}
+ */
+function formatMessageFromIdWithValues(locale, icuMessageId, values) {
```

Comment:
what about
```suggestion
function getFormattedFromIdAndValues(locale, icuMessageId, values) {
```
to match `getFormatted` (param order for `getFormatted` vs `_formatIcuMessage` is unfortunate but whatever)